docs/configure/integration/typescript.mdx

```diff
@@ -96,7 +96,7 @@ Out of the box, Storybook supports the `satisfies` operator for almost every fra
 <IfRenderer renderer={[ 'vue', 'svelte' ]}>
   ### The TypeScript auto-completion is not working on my editor
 
-  If you're using Vue single file components and TypeScript, you can add both [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) and the [TypeScript Vue Plugin](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) for editor support, additional type safety and auto-completion. Nevertheless, if you're working with Svelte, you can add the [Svelte for VSCode extension](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) for similar benefits.
+  If you're using Vue single file components and TypeScript, you can add [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) for editor support, additional type safety and auto-completion. Nevertheless, if you're working with Svelte, you can add the [Svelte for VSCode extension](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) for similar benefits.
```

Comment:
@Joe-Moran when you can, could you adjust this to the following to ensure we're referencing the correct extension name:
```suggestion
  If you're using Vue single file components and TypeScript, you can add the official [Vue](https://marketplace.visualstudio.com/items?itemName=Vue.volar) extension for editor support, additional type safety and auto-completion. Nevertheless, if you're working with Svelte, you can add the [Svelte for VSCode extension](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) for similar benefits.
```

Reply:
@Joe-Moran, one small item as well. I was wondering if you could adjust the [Writing Stories in TypeScript](https://github.com/storybookjs/storybook/blob/next/docs/writing-stories/typescript.mdx) and use the same naming to ensure consistency across the documentation. I saw the earlier pr that you opened.